_posts/patterns/2015-04-15-icons.md

```diff
@@ -624,7 +635,7 @@ info:               Icons are used to clearly and visually represent some inform
             </span>
         </div>
         <div class="icon-classname">
-            Home
+            home
         </div>
     </div>
     <div class="example example-icon">
```

Comment:
It looks like "External link" at line [649](https://github.com/edx/ux-pattern-library/pull/284/files#diff-8e6a0a47e453c2074d1a3b7fbd3c0280R649) should be updated to the "external-link" instead.